esrally/rally.py

```diff
@@ -495,27 +546,32 @@ def with_actor_system(runnable, cfg):
                 console.println(SKULL)
                 console.println("")
             elif not shutdown_complete:
-                console.warn("Could not terminate all internal processes within timeout. Please check and force-terminate all Rally processes.")
+                console.warn(
+                    "Could not terminate all internal processes within timeout. Please check and force-terminate all Rally processes.")
 
 
 def generate(cfg):
     chart_generator.generate(cfg)
 
 
+SUB_COMMANDS = {
+    "compare": reporter.compare,
+    "list": dispatch_list,
+    "race": race,
+    "download": mechanic.download,
+    "provision": mechanic.provision,
```

Comment:
I think we want to document the `provision` and `start` / `stop` commands either in this PR or in a separate PR. @danielmitterdorfer WDYT?